### DISCUSSION TITLE
Playing with drawing & SVGs

### DISCUSSION THREAD

**Discussion by wbpaley** [dyn4j/dyn4j#237] [Show and tell]
Made a module in our analytics/goofing-around environment that lets one hand-draw objects and see what they do.

Also made the module write a recording as an animated SVG; here's the first that worked.

(Note: ~6MB file size--if you have a slow link it'll take time to load)

![Dyn4jDraw](https://user-images.githubusercontent.com/60867/198853175-5e2b10e9-0927-4b24-a56a-6622139a2b12.svg)

**Comment by wbpaley** [comment]
This one has a better non-narrative "story line." 
Please pardon 8MB size. (Working on compression...)

![Dyn4jDraw](https://user-images.githubusercontent.com/60867/198936448-26c0f36a-cfde-4c70-bed6-f5de78fc2ce8.svg)

**Comment by wbpaley** [comment]
More interesting with a distance joint, especially when orbit-induced centrifugal forces act against the joint's own forces toward resting distance. The (very faint, use Ctrl-+)  arrows scale with force, pointing toward the rest distance.

If you see a black shadow trailing the fast object (I do) you're wrong. You _perceive_ it, but it's not actually there. Test that by wagging the browser around quickly on your screen--you'll see it again but with an offset related to your wagging, not Mr. Bittle's lovely animation vectors. (Chased that bug for a few minutes--but it was a bug in my own psychophysics!) [Edit: don't see it on my iPhone; may be a bug in the monitor.]

Tell me if these progress snapshots are clogging up your discussion space... 

![Dyn4jDraw](https://user-images.githubusercontent.com/60867/199609387-2e04be00-b850-4ae3-b25f-02560ef7087d.svg)

**Reply by wnbittle** [reply]
No problem at all.  I'd love it if others did the same.  While I enjoy working on the project, the fact that others have found it useful or done cool things with it is even better.  Keep it up!
